lib/tasks/elasticsearch.rake

```diff
@@ -0,0 +1,9 @@
+namespace :elasticsearch do
+  desc 'Delete the "resource" index'
+  task :delete_resource_index do
+    client = Elasticsearch::Client.new(
```

Comment:
I'm not sure I see it as correct or incorrect, but I can't explain why. I'm sure it's fine either way.

I was more just curious if you can use the same client instance for simplicity but there's really no need.